lib/device.js

```diff
@@ -55,7 +55,11 @@ const util = require('util'),
 
             function _makeSession(next) {
                 options.nReplies = 1; // -n 1
-                options.session = new novacom.Session(options.device, next);
+                if (!options.session) {
```

Comment:
I think the old code is better.
Rather than checking the session and passing it over
I think it's best to create a new one, as it's certainly best to create a new one and there may be cases where you are using an old session.

Reply:
I think that one ssh connection for one target connection is enough.
In the meantime, there were cases in which up to 3 sessions were connected.

This code is a code that prevents connecting multiple sessions.

Reply:
Keep it.